### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by prestonrodrigues**
Could you update the doc

https://github.com/prestonrodrigues/D040-Complex_Feature_Handling_Engineering_Report/blob/patch-1/7-standards-assessment.adoc

to the master branch. In addition, some files in the image folder 3dps# (# 1-6).

### CODE REVIEW

File: 7-standards-assessment.adoc [hunk 1]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+.OpenAPI 
+image::images/3dps1.png[]
+
+The **data Layer** function is used to retrieve the name of the available data layers. The current implementation supports one layer only:  _“3D_CityModel_manhattan”_ . However, in case the service supports more then one data layer it will show a list of supported data layers.
+
+----
+{“datalayer ”: “3D_CityModel_manhattan,3D_CityModel_geneva,3D_CityModel_stuttgart"}
+----
+
+To test the REST API call, first clicking on the _Try it out_  button and then _execute_ button.
```
Comment (cportele) [inline 7-standards-assessment.adoc]:
```suggestion
To test the Web API, first click on the _Try it out_  button and then the _Execute_ button.
```

File: 7-standards-assessment.adoc [hunk 2]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+
+[#img-App3]
+.OpenAPI Data Layer
+image::images/3dps2.png[]
+
+Since this call does not take any input you will see the output as shown in Fig 3. In addition, it also shows the curl request with all the necessary query parameters.
```
Comment (cportele) [inline 7-standards-assessment.adoc]: As above, it won't be Figure 3.

File: 7-standards-assessment.adoc [hunk 3]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+
+The **3D Scene** function on the other hand takes three input. The first is the path parameter(_layer_) and the remaining two are query parameters(_format_,_bbox_) see Fig 4
```
Comment (cportele) [inline 7-standards-assessment.adoc]:
```suggestion
The **3D Scene** function on the other hand takes three input parameters. The first is a path parameter (_layer_) and the remaining two are query parameters (_format_,_bbox_). See Fig. <change to the correct number>.
```

File: 7-standards-assessment.adoc [hunk 4]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+
+[#img-App4]
+.OpenAPI 3D Scene
+image::images/3dps3.png[]
+
+A Successful execution will provide the a tilesetlink (see example below)
```
Comment (cportele) [inline 7-standards-assessment.adoc]:
```suggestion
A successful execution will provide the a tileset link (see example below).
```

File: 7-standards-assessment.adoc [hunk 5]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+
+----
+tilesetLink :http://tb13.igd.fraunhofer.de:8083/3dps/datahoster/fc38b549-4989-4703-bb27-f5b4fd33fd90/manhattan/tileset.json
```
Comment (cportele) [inline 7-standards-assessment.adoc]: This should be a `link` (https://github.com/opengeospatial/WFS_FES/blob/master/core/openapi/schemas/link.yaml). Again, perhaps just add a note about the simplified approach taken in the prototype.

File: 7-standards-assessment.adoc [hunk 6]
```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
...
+This _tilesetLink_ can be rendered in the Cesium globe. To show the feasibility of our approach we have developed an intergrated  prototype see Fig 5. Please visit the following http://81.169.187.7:8081/3DPS_App/[ http://81.169.187.7:8081/3DPS_App/]. 
+
+[#img-App5]
+.3DPS APP
+image::images/3dps5.png[]
+
+To educate yourself and get use the platform please visit the you tube link https://www.youtube.com/watch?v=6EwlwAhtK7Q[https://www.youtube.com/watch?v=6EwlwAhtK7Q].
+
+
+NOTE: Multiple Layers is not yet implemented
```
Comment (cportele) [inline 7-standards-assessment.adoc]:
```suggestion
NOTE: Multiple layers are not implemented in the prototype.
```

**Comment by prestonrodrigues** [comment]
comments and suggestions updated ....

**Comment by prestonrodrigues** [comment]
@cportele 

:+1:  suggestions/comments resolved ready to merge.

**Comment by cportele** [comment]
thanks @prestonrodrigues - PR merged

**Comment by prestonrodrigues** [comment]
:clap: thanks @cportele

**Pull request merged**